test/src/code/code_test.dart

```diff
@@ -4,9 +4,11 @@ import 'package:code_text_field/src/code/code.dart';
 import 'package:code_text_field/src/code/code_line.dart';
 import 'package:collection/collection.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:highlight/highlight.dart';
+import 'package:highlight/languages/angelscript.dart';
+import 'package:highlight/languages/java.dart';
 
-const singleLineComments = ['//', '#'];
-
+final Mode language = java;
```

Comment:
We prefer inferred typing as it is more readable: `final _language = java`.
Please also make it private.
Please also make `text` private.